docs/user/convert/parsing_and_validation.rst

```diff
@@ -0,0 +1,119 @@
+Parsing and Validation
+----------------------
+
+GeoMesa provides options for dealing with bad input records that can be configured on your converter. For example,
```

Comment:
"...records that can be configured..." => "...records, which can be configured..."

(Otherwise it makes it sound like that it's the  records that are configured, not the options.)